### PR TITLE
Object tracking data

### PR DESCRIPTION
**Pull request by FrankYoungchen**
Object tracking data

### CODE REVIEW

File: docs/tutorials/datasets/det.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+"""Prepare ILSVRC 2015DET dataset
```
Comment (bryanyzhu) [inline docs/tutorials/datasets/det.py:1]: Add a space between 2015 and DET.

File: docs/tutorials/datasets/det.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+=========================================
+
+`ILSVRC DET dataset <http://image-net.org/>`_ is
+a dataset for object localization/detection and image/scene classification task.
+It contains two thousands full labeled categories for detection and one thousands categories for 1000 categories
```
Comment (bryanyzhu) [inline docs/tutorials/datasets/det.py:6]: What do you mean by "one thousands categories for 1000 categories"? Maybe a typo. I think you are trying to say 1000 categories for classification.

File: docs/tutorials/datasets/mscoco_tracking.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+"""Prepare COCO datasets
+==============================
+
+`COCO <http://cocodataset.org/#home>`_ is a large-scale object detection, segmentation, and captioning datasetself.
+This tutorial will walk through the steps of preparing this dataset for GluonCV for object tracking.
```
Comment (bryanyzhu) [inline docs/tutorials/datasets/mscoco_tracking.py:5]: for GluonCV for object tracking -> for object tracking in GluonCV

File: docs/tutorials/datasets/mscoco_tracking.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
++------------------------------------------------------------------------------------------------------------------------+--------+------------------------------------------+
+
+
+The easiest way to download and unpack these files is to download helper script and we suggest run the command because it included download dataset and data processing
+:download:`mscoco_racking.py<../../../scripts/datasets/mscoco_racking.py>` and run
+the following command:
+
+The easiest way is to run this script:
+
+ :download:`Download script: coco_tracking.py<../../../scripts/datasets/mscoco_racking.py>`
```
Comment (bryanyzhu) [inline docs/tutorials/datasets/mscoco_tracking.py:41]: racking -> tracking

File: docs/tutorials/datasets/vid.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
+"""Prepare ILSVRC 2015VId dataset
+=========================================
+
+`ILSVRC DET dataset <http://image-net.org/>`_ is
```
Comment (bryanyzhu) [inline docs/tutorials/datasets/vid.py:4]: is ? I think you need to complete this sentence.

File: docs/tutorials/datasets/vid.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
+"""Prepare ILSVRC 2015VId dataset
```
Comment (bryanyzhu) [inline docs/tutorials/datasets/vid.py:1]: 2015VId -> 2015 VID

File: docs/tutorials/datasets/youtube_bb.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
+"""Prepare Youtube_bb dataset
+=========================================
+
+`ILSVRC DET dataset <http://research.google.com/youtube_bb/>`_ is
```
Comment (bryanyzhu) [inline docs/tutorials/datasets/youtube_bb.py:4]: should be youtube_bb dataset, right? Not DET dataset.

File: docs/tutorials/datasets/youtube_bb.py [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+This tutorial helps you to download Youtube_bb and set it up for later experiments.
+
+.. hint::
+
+   You need 195G free disk space to download and extract this dataset.
+   SSD harddrives are recommended for faster speed.
+   The time it takes to prepare the dataset depends on your Internet connection
+   and disk speed. 
+
+If you want to download it and follow it <http://github.com/STVIR/pysot/tree/master/training_dataset/yt_bb>`
```
Comment (bryanyzhu) [inline docs/tutorials/datasets/youtube_bb.py:16]: If you want to download this dataset, please follow XXX.

File: scripts/datasets/otb2015.py [hunk 9]
```diff
@@ -0,0 +1,69 @@
+"""this script is used to prepare Otb2015 dataset for tracking,
+which is Single Object Tracking benchmark"""
```
Comment (bryanyzhu) [inline scripts/datasets/otb2015.py:2]: Do we need another simple tutorial about "how to prepare otb dataset"? Just like the rest datasets?

**Comment by bryanyzhu** [comment]
@zhreshold Any thoughts on this error during building doc? Thank you. 

`mxnet.base.MXNetError: [06:23:45] src/ndarray/ndarray.cc:641: Check failed: !is_view:`

**Comment by zhreshold** [comment]
> @zhreshold Any thoughts on this error during building doc? Thank you.
> 
> `mxnet.base.MXNetError: [06:23:45] src/ndarray/ndarray.cc:641: Check failed: !is_view:`

@wuxun-zhang The `is_view` error happens again, how to debug which op is causing this bug?

**Comment by wuxun-zhang** [comment]
cc @pengzhao-intel

**Comment by pengzhao-intel** [comment]
> cc @pengzhao-intel

Thanks, the team will take a look and back to the issue later.

**Comment by wuxun-zhang** [comment]
when clicking the "Details" button, we can only find the below error:
```
pylint --rcfile=/var/lib/jenkins/workspace/workspace/gluon-cv-lint/tests/pylintrc --ignore-patterns=".*\.so$,.*\.dll$,.*\.dylib$" gluoncv

************* Module gluoncv.data.tracking_data.track

gluoncv/data/tracking_data/track.py:14:0: W0611: Unused import pdb (unused-import)

gluoncv/data/tracking_data/track.py:14:0: C0411: standard import "import pdb" should be placed before "import numpy as np" (wrong-import-order)
```

Where to find the exact error `mxnet.base.MXNetError: [06:23:45] src/ndarray/ndarray.cc:641: Check failed: !is_view:`?

**Comment by zhreshold** [comment]
@wuxun-zhang https://ci.mxnet.io/blue/organizations/jenkins/gluon-cv/detail/PR-1334/5/pipeline/ the github hook is messed up

**Comment by zhreshold** [comment]
@FrankYoungchen Please try to rebase off master with #1369 merged, it can reduce the change the CI runs oom or killed

**Comment by mli** [comment]
Job PR-1334-9 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1334/9/index.html 
Code coverage of this PR:  vs. Master:

**Pull request merged**